src/plugins/recordTypes/collectionobject/forms/default.jsx

```diff
@@ -0,0 +1,236 @@
+              <Field name="objectProductionDateGroup" />
+            </Field>
+
+            <Field name="objectProductionPersonGroupList">
+              <Field name="objectProductionPersonGroup">
+                <Field name="objectProductionPerson" />
+                <Field name="objectProductionPersonRole" />
+              </Field>
+            </Field>
+          </Col>
+
+          <Col>
+            <Field name="source" subpath="ns2:collectionobjects_cinefiles" />
+            <Row>
+              <Col>
+                <Field name="numberOfObjects" />
+                <Field name="accessCode" subpath="ns2:collectionobjects_cinefiles" />
+              </Col>
+
+              <Col>
+                <Field name="pageInfo" subpath="ns2:collectionobjects_cinefiles" />
+                <Field name="recordStatus" />
+              </Col>
+            </Row>
+
+            <Field name="briefDescriptions">
+              <Field name="briefDescription" />
+            </Field>
+
+            <Field name="distinguishingFeatures" />
+
+            <Field name="comments">
+              <Field name="comment" />
+            </Field>
+          </Col>
+        </Row>
+
+        <Row>
+          <Field name="hasCastCr" subpath="ns2:collectionobjects_cinefiles" />
+          <Field name="hasTechCr" subpath="ns2:collectionobjects_cinefiles" />
+          <Field name="hasBoxInfo" subpath="ns2:collectionobjects_cinefiles" />
+          <Field name="hasFilmog" subpath="ns2:collectionobjects_cinefiles" />
+          <Field name="hasBiblio" subpath="ns2:collectionobjects_cinefiles" />
+          <Field name="hasDistCo" subpath="ns2:collectionobjects_cinefiles" />
+          <Field name="hasProdCo" subpath="ns2:collectionobjects_cinefiles" />
+          <Field name="hasCostInfo" subpath="ns2:collectionobjects_cinefiles" />
+          <Field name="hasIllust" subpath="ns2:collectionobjects_cinefiles" />
+        </Row>
+
+        <Cols>
+          <Col>
+            <Field name="docLanguages" subpath="ns2:collectionobjects_cinefiles">
+              <Field name="docLanguage" />
+            </Field>
+            <Field name="docSubjects" subpath="ns2:collectionobjects_cinefiles">
+              <Field name="docSubject" />
+            </Field>
+          </Col>
+
+          <Col>
+            <Field name="filmSubjects" subpath="ns2:collectionobjects_cinefiles">
+              <Field name="filmSubject" />
+            </Field>
+            <Field name="nameSubjects" subpath="ns2:collectionobjects_cinefiles">
+              <Field name="nameSubject" />
+            </Field>
+          </Col>
+        </Cols>
+      </Panel>
+
+      <Panel name="desc" collapsible collapsed>
+        <Row>
+          <Field name="copyNumber" />
+          <Field name="editionNumber" />
+        </Row>
+
+        {extensions.dimension.form}
+        <Row>
+          <Field name="objectStatusList">
+            <Field name="objectStatus" />
+          </Field>
+
```

Comment:
objectStatusList and colors are losing their left/right margin (a bug). For now, wrap each in a <Col>, that should fix it.